src/tls_wrap.cc

```diff
@@ -101,6 +101,15 @@ TLSWrap::~TLSWrap() {
 #ifdef SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
   sni_context_.Reset();
 #endif  // SSL_CTRL_SET_TLSEXT_SERVERNAME_CB
+
```

Comment:
I would add a comment before this block linking to the issue.

Reply:
I’ve added a reference to the test and a quick explanation, the test contains a link to the issue.